examples/app_boring/app.py

```diff
@@ -43,6 +43,10 @@ def __init__(self):
             raise_exception=True,
         )
 
+    @property
+    def ready(self) -> bool:
```

Comment:
shall this be property for the base class?

Reply:
It is already part of the LightningFlow base class